simplefi-aave-v2/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "simplefi-aave",
+  "license": "UNLICENSED",
+  "scripts": {
+    "codegen": "graph codegen",
+    "build": "graph build",
+    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ gvladika/simplefi-aave",
```

Comment:
nit: For consistency, shouldn't we name it as the folder? `simplefi-aave-v2` (same on line 2)

Reply:
Sorry I see comment only now. Good point, I will align with new naming convention